common/scala/src/main/scala/org/apache/openwhisk/core/database/Batcher.scala

```diff
@@ -64,7 +64,7 @@ class Batcher[T, R](batchSize: Int, concurrency: Int, retry: Int)(operation: (Se
       completionMatcher = cm,
       failureMatcher = PartialFunction.empty[Any, Throwable],
       bufferSize = Int.MaxValue,
-      overflowStrategy = OverflowStrategy.dropNew)
+      overflowStrategy = OverflowStrategy.dropBuffer)
```

Comment:
this is the only behavior change in the commit. however as you can see in the line above the buffer size for this stream was int maxValue so this is functionally no behavior change.